.github/workflows/ci.yml

```
@@ -27,23 +27,33 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: ["8.10.0", 8.x, 10.x, 12.x, 13.x, 14.x]
+        eslint: [6, 7]
+        node: ['8.10.0', 8, 10, 12, 14, 16]
+        exclude:
+          - eslint: 7
+            node: 8
+          - eslint: 7
+            node: 8.10
```

Sorry, I missed one occurrence:

```suggestion
            node: “8.10.0”
```